src/main/java/food/finder/foodfinder/foodgroup/FoodGroup.java

```diff
@@ -3,11 +3,8 @@
 import food.finder.foodfinder.user.User;
 import lombok.Getter;
 import lombok.Setter;
-import org.springframework.data.neo4j.core.schema.GeneratedValue;
-import org.springframework.data.neo4j.core.schema.Id;
-import org.springframework.data.neo4j.core.schema.Node;
-import org.springframework.data.neo4j.core.schema.Property;
-import org.springframework.data.neo4j.core.schema.Relationship;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.neo4j.core.schema.*;
```

Comment:
And my wild card import seems lower than yours apparently

Reply:
I would say remove this wildcard import. This is one standard that I have picked up from the IVC world

Reply:
Man I can't figure out how to get it to NOT do that XD

Reply:
Got it!